Client/Extensions/PaymentRequestExtension.swift

```diff
@@ -44,16 +44,16 @@ extension PaymentRequestExtension: TabContentScript {
     }
     
     func scriptMessageHandlerName() -> String? {
-        return PaymentRequestExtension.name()
+        return "\(PaymentRequestExtension.name())\(UserScriptManager.messageHandlerTokenString)"
     }
     
     private func sendPaymentRequestError(errorName: String, errorMessage: String) {
         ensureMainThread {
-            self.tab?.webView?.evaluateJavaScript("PaymentRequestCallback\(self.token).paymentreq_postCreate('', '\(errorName)', '\(errorMessage)')") { _, error in
+            self.tab?.webView?.evaluateSafeJavaScript(functionName: "PaymentRequestCallback\(self.token).payemntreq_postCreate", args: ["", errorName, errorMessage], completion: { _, error in
```

Comment:
no need completion keyword, can be actually used

`self.tab?.webView?.evaluateSafeJavaScript(functionName: "PaymentRequestCallback\(self.token).payemntreq_postCreate", args: ["", errorName, errorMessage])  { _, error in`

Reply:
I get a syntax error when I remove the completion keyword.

Reply:
It is a completion block in swift , you can consider this like a lambda, try using it like 
```
    private func sendPaymentRequestError(errorName: String, errorMessage: String) {
        ensureMainThread {
            self.tab?.webView?.evaluateSafeJavaScript(functionName: "PaymentRequestCallback\(self.token).payemntreq_postCreate", args: ["", errorName, errorMessage]) { _, error in
                    if error != nil {
                        log.error(error)
                    }
            }
        }
    }
```

you will not get an error

Reply:
And I believe the method name of js function is used wrong, as far as I sa inside PaymentRequest.js the property name is `paymentreq_postCreate`
it is not `payemntreq_postCreate`